code/modules/clothing/outfits/ert.dm

```diff
@@ -6,6 +6,8 @@
 	shoes = /obj/item/clothing/shoes/combat/swat
 	gloves = /obj/item/clothing/gloves/combat
 	ears = /obj/item/radio/headset/headset_cent/alt
+	implants = list(/obj/item/implant/mindshield/implant)
+	implants = list(/obj/item/organ/cyberimp/eyes/hud/security)
```

Comment:
```suggestion
	implants = list(/obj/item/implant/mindshield, /obj/item/organ/cyberimp/eyes/hud/security)
```

Reply:
When I do that the hud  doesn't work
![image](https://user-images.githubusercontent.com/82665345/180621648-4bcb8a2e-9121-4c09-804c-db6e03dde699.png)
Doesn't even show in the brain
![image](https://user-images.githubusercontent.com/82665345/180621658-6979385a-8adc-4973-9e95-d3b11d9eaf56.png)

Reply:
Nevermind I was dumb...

Reply:
Fixed